color.js

```diff
@@ -90,7 +90,7 @@ var color = {
 			
 			// Make sure it is always the right length
 			if(color[i].length === 1) {
-				color[i] = color[i] + color[i];
+				color[i] = "0" + color[i];
```

Comment:
Feel free to modify this PR :) you should be able to add commits to it. Gonna write a post about what it feels like when some of your earliest stuff pops up out of nowhere :D I'll probably use this as a prompt to clean this project up, might as well.

Thanks!